pkg/apis/externaldns/types.go

```diff
@@ -304,6 +306,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 
 	app.Flag("namespace", "Limit sources of endpoints to a specific namespace (default: all namespaces)").Default(defaultConfig.Namespace).StringVar(&cfg.Namespace)
 	app.Flag("annotation-filter", "Filter sources managed by external-dns via annotation using label selector semantics (default: all sources)").Default(defaultConfig.AnnotationFilter).StringVar(&cfg.AnnotationFilter)
+	app.Flag("label-filter", "Filter sources managed by external-dns via label selector when listing all resources").Default(defaultConfig.LabelFilter).StringVar(&cfg.LabelFilter)
```

Comment:
Can the help text be updated to indicated that this CLI option only works with the CRD source? Right now the help text seems imply this would work with all sources.

Reply:
Fixed, I also rebased